src/mods/vr/runtimes/OpenXR.cpp

```diff
@@ -983,12 +983,19 @@ std::optional<std::string> OpenXR::initialize_actions(const std::string& json_st
             }
         }
 
-        auto filename = controller + ".json";
+        auto profile_file = controller + ".json";
 
         // replace the slashes with underscores
-        std::replace(filename.begin(), filename.end(), '/', '_');
+        std::replace(profile_file.begin(), profile_file.end(), '/', '_');
+
+        // If the json exists in the game's profile dir, use that.    
+        auto filename = (Framework::get_persistent_dir() / profile_file).string();
 
-        filename = (Framework::get_persistent_dir() / filename).string();
+        // If not, check for global profile in UEVR\Profiles dir
+        if (!std::filesystem::exists(filename)) {
+          filename = (Framework::get_persistent_dir() / ".." / "UEVR" / "Profiles" / profile_file).string();
+          spdlog::info("[VR] Setting bindings file to {}", filename);
```

Comment:
inconsistent two-space tabbing compared to four for the rest of the project.

Reply:
Ok let me see if I can enable white space characters in my editor and figure that out.

Reply:
I think I updated my pull request (sorry, new to github)